tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -898,6 +909,98 @@ def fit_extra(self, hyperparameters, time_limit=None, base_model_names=None, **k
         self.save()
         return self
 
+
+    def _filter_pseudo(self, y_pred_proba_og, min_percentage: float = 0.05, max_percentage: float = 0.6,
```

Comment:
will be good to have 1 line of documentation describing what each method is intended to do.